genesyscloud/resource_genesyscloud_routing_skill_group_test.go

```diff
@@ -0,0 +1,96 @@
+package genesyscloud
+
+import (
+	"encoding/json"
+	"fmt"
+	"os"
+	"testing"
+
+	"github.com/google/uuid"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/terraform"
+	"github.com/mypurecloud/platform-client-sdk-go/v72/platformclientv2"
+)
+
+func TestAccResourceRoutingSkillGroupBasic(t *testing.T) {
+	var (
+		skillGroupResource = "testskillgroup1"
+		skillGroupName     = "SkillGroup" + uuid.NewString()
+	)
+
+	resource.Test(t, resource.TestCase{
+		PreCheck:          func() { testAccPreCheck(t) },
+		ProviderFactories: providerFactories,
+		Steps: []resource.TestStep{
+			{
+				// Create
+				Config: generateRoutingSkillGroupResource(
+					skillGroupResource,
+					skillGroupName,
+				),
+				Check: resource.ComposeTestCheckFunc(
+					resource.TestCheckResourceAttr("genesyscloud_routing_skill_group."+skillGroupResource, "name", skillGroupName),
+				),
+			},
+			{
+				// Import/Read
+				ResourceName:      "genesyscloud_routing_skill_group." + skillGroupResource,
+				ImportState:       true,
+				ImportStateVerify: true,
+			},
+		},
+		CheckDestroy: testVerifySkillsDestroyed,
```

Comment:
Guessing you mean to call `testVerifySkillGroupDestroyed` here

Reply:
Done

Reply:
Code by cut and paste has its risks